packages/webdriver/src/utils.js

```diff
@@ -22,8 +22,14 @@ export function isSuccessfulResponse (statusCode, body) {
      * response contains a body
      */
     if (!body || typeof body.value === 'undefined') {
```

Comment:
I think overall condition can be improved to avoid nested conditions

Reply:
I thought about that, and my first local copy is in one line.
splitting it to nested condition would make easy to read. Otherwise I have to write test case on it to provide more context.